src/challenges/challengeOne/1.Accordion/Accordion.test.tsx

```diff
@@ -1,8 +1,31 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Accordion } from "./Accordion";
-
-// describe("Accordion", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Accordion, AccordionProps } from "./Accordion";
+
+describe("Accordion", () => {
+  const renderComponent = (additionalProps?: any) => {
```

Comment:
Could have added the partial props here

```suggestion
  const renderComponent = (additionalProps?: Partial<AccordionProps>) => {
```